debian/changelog

```diff
@@ -1,6 +1,11 @@
+sonic-rest-api-server (1.0.1) stable; urgency=medium
```

Comment:
This results in a package named sonic-rest-api_1.0.1_amd64.deb, we need to change the Dockerfiles to reflect the new package name for the build to succeed